pom.xml

```diff
@@ -302,6 +303,31 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>flatten-maven-plugin</artifactId>
+				<version>1.6.0</version>
+				<configuration>
+					<outputDirectory>${project.build.directory}</outputDirectory>
+					<flattenMode>oss</flattenMode>
+				</configuration>
+				<executions>
+					<execution>
+						<id>flatten</id>
+						<phase>process-resources</phase>
+						<goals>
+							<goal>flatten</goal>
+						</goals>
+					</execution>
+					<execution>
+						<id>flatten.clean</id>
+						<phase>clean</phase>
+						<goals>
+							<goal>clean</goal>
```

Comment:
Not sure what you mean, but indeed don’t need `clean` here at all since I made the flatten result be in the `target` dir, which is cleaned by standard goal. Plus that would make no need for the gitignore as well.